### ISSUE TITLE
Get initial data for app

### ISSUE THREAD

**Issue by molly-clabots** [molly-clabotst/lmnop#3]
How will this application get initial data? At the moment, shows, venues, and artists must be added through the administrator console. Should users be able to add venues, artists, shows? If so, how will you moderate this? Where else will your application get data from? Maybe you could focus on the Twin Cities and find a list of venues; and figure out some programatic method of getting data for shows, perhaps write a web scraper for First-Avenue.com and other popular venues?
Should users be able to create shows from artists and venues already in the database?

**Comment by Di30an** [comment]
Went ahead a worked on this. I am for the option of not allowing users to add info at this point, stressing the minimum viable product. Here is a link to a program I wrote to scrape info from either ticketmaster or from first ave's website. Ticket Master uses their API so you will need one, however the first ave one is just plain old HTLM scrapping. Not sure what data structure we would like to use on this so I just created a list. Working on scraping the image from the first ave website aswell. 

https://github.com/Di30an/Ticketmaster_First_Ave_Info_Scrapper
